.github/workflows/crowdin_sync.yml

```diff
@@ -8,6 +8,7 @@ concurrency:
 
 jobs:
   crowdin-sync:
+    if: github.repository == 'trezor/trezor-suite'
```

Comment:
It can be useful to have crowdin-sync allowed in private repo, but it can be done manually if needed.